packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -55,10 +56,20 @@
     Union[list[Any], dict[str, Any]],  # arrays | object
 ]
 
-InputID = OutputID = constr(regex=PROPERTY_KEY_RE)
-InputsDict = dict[InputID, InputTypes]
-OutputsDict = dict[OutputID, OutputTypes]
-UnitStr = constr(strip_whitespace=True)
+
+class KeyIDStr(ConstrainedStr):
+    regex = re.compile(PROPERTY_KEY_RE)
+
+
+InputID: TypeAlias = KeyIDStr
```

Comment:
Question: adding the ```TypeAlias``` type here, does it bring something? just wondering.

Reply:
yes. It helps mypy in https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases